AsyncDisplayKit/ASMapNode.mm

```diff
@@ -392,21 +392,14 @@ - (void)setSnapshotSizeWithReloadIfNeeded:(CGSize)snapshotSize
 
 - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 {
-  CGSize size = self.preferredFrameSize;
-  if (CGSizeEqualToSize(size, CGSizeZero)) {
-    size = constrainedSize;
-    
-    // FIXME: Need a better way to allow maps to take up the right amount of space in a layout (sizeRange, etc)
-    // These fallbacks protect against inheriting a constrainedSize that contains a CGFLOAT_MAX value.
-    if (!isValidForLayout(size.width)) {
-      size.width = 100.0;
-    }
-    if (!isValidForLayout(size.height)) {
-      size.height = 100.0;
-    }
+  // FIXME: Need a better way to allow maps to take up the right amount of space in a layout (sizeRange, etc)
+  // These fallbacks protect against inheriting a constrainedSize that contains a CGFLOAT_MAX value.
+  if (!ASCGSizeIsValidForLayout(constrainedSize)) {
```

Comment:
Optional: This method was a bit strange to read, are there any clearer alternatives to `ASCGSizeIsValidForLayout`?